plugin/gradle/libs.versions.toml

```diff
@@ -5,21 +5,21 @@ androidPlugin = "4.1.0"
 semver = "1.1.1"
 jgit = "5.6.0.201912101111-r"
 sl4fj = "1.7.30"
-gradleWrapper = "7.1.1"
-gradleWrapperSha = "33ad4583fd7ee156f533778736fa1b4940bd83b433934d1cc4e9f608e99a6a89"
+gradleWrapper = "7.2"
+gradleWrapperSha = "f581709a9c35e9cb92e16f585d2c4bc99b2b1a5f85d2badbd3dc6bff59e1e6dd"
 junit5 = "5.5.2"
 assertJ = "3.11.1"
 commonsIo = "2.8.0"
 archUnit = "0.16.0"
 
 [libraries]
-android-gradle-plugin = { module = "com.android.tools.build:gradle", version.ref = "androidPlugin" }
+android-gradle = { module = "com.android.tools.build:gradle", version.ref = "androidPlugin" }
```

Comment:
Why do you removed `-plugin` suffix here and for kotlin plugins?

Reply:
```
org.gradle.api.InvalidUserDataException: Invalid catalog definition:
  - Problem: In version catalog libs, alias 'kotlin-gradle-plugin' is not a valid alias.

* Try:
Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Exception is:
java.lang.RuntimeException: org.gradle.api.InvalidUserDataException: Invalid catalog definition:
  - Problem: In version catalog libs, alias 'kotlin-gradle-plugin' is not a valid alias.
    
    Reason: It shouldn't end with 'plugin'.
    
    Possible solution: Use a different alias which doesn't end with 'bundle', 'bundles', 'plugin', 'plugins', 'version' or 'versions'.
    
    Please refer to https://docs.gradle.org/7.2/userguide/version_catalog_problems.html#reserved_alias_name for more details about this problem.
```

Reply:
Put plugins into `plugins` block should be a good sulotion, like [KotlinPoet did](https://github.com/square/kotlinpoet/blob/0d1030d6c71d41213555b560868604a4bfd23361/gradle/libs.versions.toml#L20-L25).